06-blockchain/02-contracts.md

```diff
@@ -0,0 +1,4 @@
+## NeoFS Smart Contracts
+
+This section contains brief description about the role of contracts and
+autogenerated documentation from source code.
```

Comment:
The note about auto-generated nature of the following chapter doesn't add any good, let's omit it.